specifications/negotiation/contract.negotiation.binding.https.md

```diff
@@ -80,11 +78,9 @@ containing the [Contract Negotiation](#ack-contract-negotiation):
 Predefined states are: `REQUESTED`, `OFFERED`, `ACCEPTED`, `AGREED`, `VERIFIED`, `FINALIZED`, and `TERMINATED` (
 see [[[#contract-negotiation-states]]]).
 
-### The `negotiations/request` Endpoint _(Provider-side)_
-
-#### POST {#negotiations-request-post}
+### Contract Request Endpoint (init) {#negotiations-request-post}
```

Comment:
don't understand the `(init)` part. Is there a more expressive way to show the difference to `{#negotiations-providerpid-request-post}`?